Tools/to_festival_plan.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+declare -r festival_year=$(basename $(dirname $(pwd)))
+declare -r festival=${festival_year:0:4}
+declare -r year=${festival_year:4:4}
+declare -r festival_dir=/Users/maartenroos/Documents/Film/$festival/${festival}${year}
+declare -r source_dir=$festival_dir/_planner_data
+declare -r source_files="$source_dir/*.csv $source_dir/*.xml"
+declare -r target_dir=$festival_dir/FestivalPlan
+declare -r backup_dir=$target_dir/_bak
+declare -r temp_dir=$target_dir/newbak
+
+function confirm() {
+    local -r question=$1
+    local answer=
+    printf "\n$question [ confirm ] "
+    read answer
+    if [  -n "$answer"  ]; then
+        comment "Not confirmed"
+        exit
+    fi
+}
+
+if [ ! -d $source_dir ]; then
+    echo "Source directory $source_dir not found."
+    exit 1
+fi
+
+if [ ! -d $target_dir ]; then
+    echo "Target directory $target_dir not found."
+    exit 2
+fi
+
+if [ ! -d $backup_dir ]; then
+    confirm "Create new backup directory $backup_dir?"
+    mkdir $backup_dir
+fi
+
+echo "Source files:"
+ls -lt $source_files
+
+pushd $target_dir
+mkdir $temp_dir
+mv $backup_dir $temp_dir
```

Comment:
Please consider to facilitate limiting the recursion level of backup directories.

Reply:
I will do that as soon as I find myself wanting to cleanup the recursive `_bak` directories.